granary/test/testdata/article_with_reposts.mf2.json

```diff
@@ -17,7 +17,7 @@
         "repost-of": ["http://example.com/abc"]
       }
     }, {
-      "type": ["h-entry", "h-as-repost"],
+      "type": ["h-cite", "h-as-share"],
```

Comment:
my understanding is `h-as-*` are experimental and intentionally not documented.

but if the ActivityStreams objectType is "share", I thought the mf2-to-activitystreams shim should be "share" too.